daemon/dockerize.py

```diff
@@ -182,6 +195,7 @@ def run(
                 ports=ports,
                 detach=True,
                 command=command,
+                entrypoint=entrypoint,
```

Comment:
Wouldn't `entrypoint=None` invalidate the default entrypoint for mini-jinad based containers?

Reply:
I think nothing will happen if that is None and the regular entrypoint will be used